readme.md

```diff
@@ -19,14 +19,6 @@ Nix configuration with home manager.
 6. Clone this repo in `$HOME/.config/home-manager`.
 7. `home-manager switch --flake .`
 
-## Commands
-
-`home-manager switch --flake .` - Evaluates the home manager configuration and provisions the specified packages in flake.nix.
-
-`nix-collect-garbage -d` - Clears up previous generations generated by home-manager switch.
```

Comment:
Remember to expire home-manager generations too (see https://github.com/Lillecarl/nixos/blob/master/clean)